pkg/sys/fs/quota/quota_linux.go

```diff
@@ -66,39 +66,43 @@ func setProjectQuota(blockFile string, projectID uint32, quota FSQuota) error {
 
 	hardLimitBlocks := uint64(math.Ceil(float64(quota.HardLimit) / float64(blockSize)))
 	softLimitBlocks := uint64(math.Ceil(float64(quota.SoftLimit) / float64(blockSize)))
```

Comment:
`const blockSize` doesn't have type and no need to typecast to `float64`. You could fix this everywhere.